homeassistant/components/swiss_public_transport/sensor.py

```diff
@@ -42,10 +43,10 @@ async def async_setup_entry(
     """Set up the sensor from a config entry created in the integrations UI."""
     coordinator = hass.data[DOMAIN][config_entry.entry_id]
 
-    name = config_entry.title
+    unique_config_title = config_entry.title
```

Comment:
Downside is, title is mutable by users, so we do not want to use this. Can you use `config_entry.unique_id`? that's nullable by default so you also need an assert to make sure it exists for type checking

Reply:
It what cases can it be Null? What should be done in that case? Is this only relevant for yaml config?

Reply:
No because in the import we set the unique_id. Like, the type of `ConfigEntry.unique_id` is just nullable by default but in this case its always filled